templates/patterns/toasts/pattern-toasts.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Toasts pattern.
+ */
+#}
+{% set wrapper_aria_live = wrapper_aria_live|default('assertive') %}
+{% set toasts_attributes = toasts_attributes.addClass(['show'])  %}
```

Comment:
This toast attributes exists previously? If not I think that must create and then add the attribute.